osu.Game/Screens/BackgroundScreen.cs

```diff
@@ -5,12 +5,15 @@
 using osu.Framework.Screens;
 using osu.Framework.Graphics;
 using osu.Framework.Input.Events;
+using osu.Game.Graphics.Backgrounds;
 using osuTK;
 
 namespace osu.Game.Screens
 {
     public abstract class BackgroundScreen : Screen, IEquatable<BackgroundScreen>
     {
+        protected Background Background;
```

Comment:
This can't just exist here like this. Properties in a class should be used by the class itself. Either `BackgroundScreen` should be responsible for setting this or it should be added in a per-class basis. Otherwise there's little point to this existing.